src/run/regional-blob-store.cts

```diff
@@ -1,8 +1,23 @@
 import { getDeployStore, GetWithMetadataOptions, Store } from '@netlify/blobs'
+import { LRUCache } from 'lru-cache'
+
+import { type BlobType, estimateBlobSize } from '../shared/cache-types.cjs'
+
+import { getRequestContext } from './handlers/request-context.cjs'
+import { getTracer } from './handlers/tracer.cjs'
+
+// lru-cache types don't like using `null` for values, so we use a symbol to represent it and do conversion
+// so it doesn't leak outside
+const NullValue = Symbol.for('null-value')
+type BlobLRUCache = LRUCache<string, BlobType | typeof NullValue | Promise<BlobType | null>>
 
 const FETCH_BEFORE_NEXT_PATCHED_IT = Symbol.for('nf-not-patched-fetch')
+const IN_MEMORY_CACHE_MAX_SIZE = Symbol.for('nf-in-memory-cache-max-size')
+const IN_MEMORY_LRU_CACHE = Symbol.for('nf-in-memory-lru-cache')
 const extendedGlobalThis = globalThis as typeof globalThis & {
   [FETCH_BEFORE_NEXT_PATCHED_IT]?: typeof globalThis.fetch
+  [IN_MEMORY_CACHE_MAX_SIZE]?: number
+  [IN_MEMORY_LRU_CACHE]?: BlobLRUCache | null
```

Comment:
Move `noOpInMemoryCache` here? Instead of `null`?

Reply:
I realized these are types, but the intention of my comment still sort of stands. Can we try to have these be non-nullable if possible?

Reply:
`noOpInMemoryCache` is not implementing same interface as `BlobLRUCache` so it couldn't be used here. To make it non-nullable I would need to create a shim class that implements `LRUCache` interface and this doesn't seem worth doing and maintaing to get rid of `null` case to me?